### PR TITLE
[ONNX] Update onnx landing page for 1.3

### PR DESCRIPTION
**Pull request by BowenBao**
* Update supported operator list.
* Update FAQ on implicit scalar casting. Traced models are now more robust.

cc @spandantiwari @lara-hdr @neginraoof Please feel free to add any missing points. Thank you!

cc @houseroad for review.

### CODE REVIEW

**Review by houseroad — APPROVED** [review]
Looks good.

File: docs/source/onnx.rst [hunk 1]
```diff
@@ -339,13 +339,15 @@ The following operators are supported:
...
 * ceil
 * clamp
 * clamp_max
 * clamp_min
 * concat
 * cos
+* cumsum
 * dim_arange
 * div
 * dropout
```
Comment (lara-hdr) [inline docs/source/onnx.rst]: do we want to add empty and empty_like?

**Comment by lara-hdr** [comment]
Should we mention the support of dictionaries with its limitation? (maybe in the section "limitations")

**Review by spandantiwari — APPROVED** [review]
LGTM.

File: docs/source/onnx.rst (hunk 1 above)
Comment (BowenBao) [reply docs/source/onnx.rst]: good catch!

**Comment by BowenBao** [comment]
> Should we mention the support of dictionaries with its limitation? (maybe in the section "limitations")

Added in the limitations section. Please review if it is accurate. Thanks!

**Review by lara-hdr — APPROVED** [review]

**Comment by houseroad** [comment]
A bit late for the doc of 1.3, but we live on the master :-)

**Pull request closed without merging**